pkg/controller/jobframework/setup.go

```diff
@@ -35,6 +38,13 @@ var (
 	errFailedMappingResource = errors.New("restMapper failed mapping resource")
 )
 
+var (
+	// APIRetryInterval is the duration to wait before retrying the API request.
+	APIRetryInterval = 20 * time.Second
+	// APIRetryMutex is the mutex used to safely access the APIRetryInterval.
+	APIRetryMutex sync.Mutex
```

Comment:
This mutex does not seem needed as the interval cannot be changed at runtime, just in tests.